### ISSUE TITLE
[Bugs] Saat di scroll next

### ISSUE THREAD

**Issue by balitax** [SeptiyanAndika/ViewPager---Swift#5]
keren mas libraryne. ada yang aneh, mungkin bugs, saat saya scroll pagernya dia bulletnya dan pagernya kembali ke sebelumnya. dan saat tak scroll previous dia juga kembali ke previous terus balik next lagi

![bugs](https://cloud.githubusercontent.com/assets/1490342/18192818/ada57c62-7101-11e6-9b3f-df259488d19c.gif)

**Comment by SeptiyanAndika** [comment]
di check dulu mas, makasih atas feedbacknya

**Comment by r3dsm0k3** [comment]
```
 public func scrollViewDidScroll(scrollView: UIScrollView) {
        NSObject.cancelPreviousPerformRequestsWithTarget(scrollView)
        self.performSelector(#selector(self.scrollViewDidEndScrollingAnimation(_:)), withObject: scrollView, afterDelay: 0.3)
    }
    
    public func scrollViewDidEndScrollingAnimation(scrollView: UIScrollView) {
        NSObject.cancelPreviousPerformRequestsWithTarget(self)
        var pageNumber = round(scrollView.contentOffset.x / scrollView.frame.size.width)
        pageNumber = pageNumber + 1
        pageControl.currentPage = Int(pageNumber)
        currentPosition = pageControl.currentPage
        scrollToPage(Int(pageNumber))
    }
```
Put that inside the `UIScrollViewDelegate` and it should fix the issue

**Comment by r3dsm0k3** [comment]
I have added a pull request. Thanks.

**Comment by SeptiyanAndika** [comment]
thanks @r3dsm0k3 .

**Comment by SeptiyanAndika** [comment]
closed by #9
